### ISSUE TITLE
Missing License

### ISSUE THREAD

**Issue by line0** [gtk2k/Unity-WebVR-Test-Assets#31]
I would like to use your template and assets in a own project, however I (or anybody else for that matter) can't because no license is provided for any portion of the provided code. This is something you probably want to fix before the list of contributors grows any further. As it stands, nobody, not even you, can use the whole code base in any way until all 4 contributors agree on a license.

**Comment by XanNava** [comment]
Yes please add a License to this. or sell it on unity asset store. I would pay money for this.
